src/web/src/asset_actions_GET.ecpp

```diff
@@ -74,68 +76,47 @@ bool database_ready;
     }
     // end sanity checks
 
-    // connect to malamute
-    MlmClientGuard client(mlm_client_new( ));
-    if (!client) {
-        std::string err = "mlm_client_new() failed.";
-        log_error (err.c_str ());
-        http_die ("internal-error", TRANSLATE_ME (err.c_str ()).c_str ());
-    }
-
-    std::string client_name = utils::generate_mlm_client_id ("web.asset_get");
-    log_debug ("malamute client name = '%s'.", client_name.c_str ());
-
-    int rv = mlm_client_connect (client, MLM_ENDPOINT, 1000, client_name.c_str ());
-    if (rv == -1) {
-        std::string err = "mlm_client_connect() failed.";
-        log_error (err.c_str ());
-        http_die ("internal-error", TRANSLATE_ME (err.c_str ()).c_str ());
-    }
-    // end connect to malamute
-
-    // send request
-    zmsg_t *rqst = zmsg_new();
-    zmsg_addstr(rqst, "GET_COMMANDS");
-    zmsg_addstr(rqst, "");
-    zmsg_addstr(rqst, checked_id.c_str());
+    auto msgbus = std::unique_ptr<messagebus::MessageBus>(messagebus::MlmMessageBus(MLM_ENDPOINT, messagebus::getClientId("tntnet")));
+    msgbus->connect();
 
-    if (mlm_client_sendto(client, "fty-nut-command", "power-actions", nullptr, 5000, &rqst) != 0) {
-        std::string err = "Failed to send request to fty-nut-command.";
-        log_error (err.c_str ());
-        http_die ("internal-error", TRANSLATE_ME (err.c_str ()).c_str ());
-    }
+    dto::commands::GetCommandsQueryDto queryDto;
+    queryDto.asset = checked_id;
 
-    // wait for reply
-    ZpollerGuard poller(zpoller_new(mlm_client_msgpipe(client), nullptr));
-    if (!zpoller_wait(poller, 5000)) {
-        std::string err = "zmq_poll() failed.";
-        log_error (err.c_str ());
-        http_die ("internal-error", TRANSLATE_ME (err.c_str ()).c_str ());
-    }
+    messagebus::Message msgRequest;
```

Comment:
Could be cool to factorize this code (sendRequest(......) like https://github.com/42ity/fty-srr-rest/blob/master/src/fty_srr_helpers.cc